Lib/asyncio/taskgroups.py

```diff
@@ -10,7 +10,22 @@
 
 
 class TaskGroup:
+    """An asynchronous context manager for managing groups of tasks.
 
+    Example use:
+
+        async with asyncio.TaskGroup() as group:
+            task1 = group.create_task(some_coroutine(...))
+            task2 = group.create_task(other_coroutine(...))
+
+        print("Both tasks have completed now.")
+
+    All tasks are awaited when the context manager exits.
+
+    Any exceptions other than asyncio.CancelledError's raised within
```

Comment:
The grammar here isn't quite right. Maybe just:

```suggestion
    Any exceptions other than asyncio.CancelledError raised within
```